ludwig/schema/features/utils.py

```diff
@@ -35,8 +35,18 @@ def update_encoders(feature_props, feature_type):
         None
     """
     for key in get_encoder_classes(feature_type):
```

Comment:
So I actually removed this in the Encoder Refactor PR since I was able to change the import structure to support all registrations. I can see an argument for keeping this just in case as a fail safe so I'm ok keeping this in though I believe it may be unnecessary.

Reply:
SGTM.